src/clients/java/src/client.zig

```diff
@@ -110,21 +115,59 @@ const ReflectionHelper = struct {
         assert(env.hasPendingException());
     }
 
-    pub fn buffer(env: *jui.JNIEnv, this_obj: jui.jobject) ?[]u8 {
+    pub fn get_send_buffer(env: *jui.JNIEnv, this_obj: jui.jobject) ?[]u8 {
```

Comment:
This doesn't need to be `pub` I think? `pub` is for visibility across files, and we only use this method in this file.

Reply:
I didn't know about that :+1:, but  I think we should keep using `pub` because it makes it visible that this function is part of the API used by other types and not an internal implementation.